demo/build.gradle

```diff
@@ -116,29 +81,8 @@ android {
     flavorDimensions 'api'
 
     productFlavors {
-        r2_0_4 {
-            dimension  'api'
-        }
-        r2_1_1 {
-            dimension  'api'
-        }
-        r2_2_0 {
-            dimension  'api'
-        }
-        r2_3_1 {
-            dimension  'api'
-        }
-        r2_4_1 {
-            dimension  'api'
-        }
-        r2_5_1 {
-            dimension  'api'
-        }
-        r2_6_0 {
-            dimension  'api'
-        }
         r2_7_3 {
-            dimension  'api'
+            dimension 'api'
         }
         r2_8_0 {
             dimension  'api'
```

Comment:
Note for Justin - just remove this space

Reply:
Done